pkgs/cli/src/main.zig

```diff
@@ -35,6 +36,12 @@ const ZeamArgs = struct {
         beam: struct {
             help: bool = false,
             mockNetwork: bool = false,
+            metricsPort: u16 = 9667,
+        },
+        generate_prometheus_config: struct {
```

Comment:
can you also move remove this out for now and add this to a utils cli package that we will be adding

Reply:
so FYI I have moved that back in, I think it's fine to have that in there for now. Let's go through pq-devnet-0 and improve once we have launched it.